src/app/components/MediaPlayer/AudioLoader/index.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { node } from 'prop-types';
+import styled from '@emotion/styled';
+import { keyframes } from '@emotion/react';
+
+const Wrapper = styled.div`
+  position: relative;
+`;
+
+const Overlay = styled.div`
+  .no-js & {
+    display: none;
+  }
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+`;
+
+const spin = keyframes`
+0% {
+  transform: rotate(0deg);
+}
+100% {
+  transform: rotate(360deg);
+}
+`;
+
+const Spinner = styled.div`
+  width: 5rem;
+  height: 5rem;
+  border-radius: 50%;
+  border: 0.3rem solid rgba(0, 0, 0, 0.2);
+  border-left-color: rgba(0, 0, 0, 0.8);
+  animation: ${spin} 1.1s infinite linear;
+`;
+
+const AudioLoader = ({ children }) => {
+  return (
+    <Wrapper>
+      <Overlay data-testid="audio-loader-overlay" aria-hidden="true">
+        <Spinner />
```

Comment:
Does the spinner never disappear? Think it should be removed from the DOM when the player loads? Just think it might be better for performance to not have a perpetually animating element in the DOM

Reply:
Can do - this still the case even if we stop animating after 3 cycles?  Not sure what's worse for performance - having a hidden element there, or updating the DOM to remove it

Reply:
Yeh good point. Not sure. What about a display none onload?

Reply:
I just had a thought that if we didn't remove the spinner then we'd always see it with CSS disabled

Reply:
Previous way was the same way we do the video placeholders - it's fine with CSS disabled - without CSS, it's just a 0-height div that sits below the player

With the new SVG way though, we need to remove the element on load

Reply:
SMP doesn't support disabled CSS anyway though - on Live, without CSS, there's just a blank space where the player should be.  It's the same on Sounds